src/store/reducers/index.js

```diff
@@ -0,0 +1,8 @@
+import { combineReducers } from 'redux';
+import foodReducer from './food';
+
+console.log('food red :', foodReducer);
+const store = combineReducers({
+  reducer: foodReducer,
```

Comment:
You should have just left here `foodReducer`  (ES2015 notation) instead of `reducer: foodReducer`. You will have always more than one reducer and it shouldn’t have name reducer.

Reply:
Change 6 line code